packages/discord-config-ui/discord_configure.js

```
@@ -0,0 +1,8 @@
+Template.configureLoginServiceDialogForDiscord.helpers({
+  siteUrl: () => Meteor.absoluteUrl(),
+});
+
+Template.configureLoginServiceDialogForDiscord.fields = () => [
+  {property: 'clientId', label: 'Client ID'},
```

ESLint

[error] Replace `property:·'clientId',·label:·'Client·ID'` with `·property:·'clientId',·label:·'Client·ID'·`

(prettier/prettier)